src/pages/login/__tests__/Login.tsx

```diff
@@ -9,7 +9,9 @@ jest.mock('../../../commons/utils/Constants', () => {
   return {
     __esModule: true,
     default: {
-      authProviders: new Map([['luminus', { name: 'LumiNUS' }]])
+      authProviders: new Map([['luminus', { name: 'LumiNUS' }]]),
+      defaultSourceChapter: 4,
+      defaultSourceVariant: 'default'
```

Comment:
Possible to use the enums here instead?

Reply:
I'm not too familiar with Jest, so I don't know if it's truly impossible, but I tried using the enums at first, and it gave me an error saying that we can't reference external variables (including enums) in Jest mocks. I didn't think it was absolutely necessary to use the enums as this was a test file.

Reply:
Ah I see. :(